### PR TITLE
Added r-constrictive as a package

### PR DESCRIPTION
**Pull request by thewchan**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by thewchan** [comment]
@conda-forge/r @conda-forge/staged-recipes This is ready for review; not sure why the Win build failed, but this in noarch anyway

**Comment by thewchan** [comment]
@conda-forge/help-r Sorry I think I might have pinged the wrong team. Review appreciated

File: recipes/r-constructive/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-constructive
+  version: {{ version|replace("-", "_") }}
+
+source:
+
+  git_url: https://github.com/cynkra/constructive
+  git_tag: v0.0.1
```
Comment (dbast) [inline recipes/r-constructive/meta.yaml:13]: Tags are mutable and by that insecure. Please instead use:
```suggestion
  url: https://github.com/cynkra/constructive/archive/refs/tags/v{{ version }}.tar.gz
  sha256: ...
```

**Comment by thewchan** [comment]
@dbast ready again!

**Comment by dbast** [comment]
@thewchan I merged all your CRAN package PRs. Your current remaining R PRs are package which are not on CRAN and I am hesitating to merge those:
* would be nice if those packages would be contributed to CRAN first, as they do extensive QA testing, e.g. check memory usage, dependency constrains, R versions compatibilities etc.
* they all have very low version numbers, like 0.0.1, which makes it unclear how reliable those packages are and how good they are maintained longterm 
* they are all noarch packages, so you can create your own channel on anaconda.org (free) and build and upload those packages there for a wider audience
* ... though there are exceptions: r-choroplethrzip is maybe an exception as it already has higher version numbers and it is not on CRAN only due size constrains according to their README.

**Comment by thewchan** [comment]
@dbast Understood, I suppose it's totally up to you all if you want to merge them or not. Appreciate the review nonetheless

**Comment by thewchan** [comment]
@dbast The more I think about this the more I'm a little confused... The review process on PyPI is no where as convoluted as in on CRAN, and we support porting over very early version numbers Python package on Conda Forge all the time. Can you help me understand why for R packages the standards are so different on Conda Forge?

**Pull request closed without merging**